jax/experimental/compilation_cache/file_system_cache_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+from file_system_cache import FileSystemCache
+import jax.test_util as jtu
+import tempfile
+
+class FileSystemCacheTest(jtu.JaxTestCase):
+
+  def test_new_directory_and_path(self):
+   with tempfile.TemporaryDirectory() as tmpdir:
+     file_system_cache_instance = FileSystemCache(tmpdir)
+     self.assertEqual(file_system_cache_instance.get("nonExistantFile"), None)
```

Comment:
This is super minor, but instead of using "nonExistentFile", use "nonExistentKey". Even though it doesn't make a difference to the test which nonexistent key is used, I think it's better to use cache-related terms instead of filesystem-related terms when testing `get` and `put`, since this test file doesn't really deal with files directly (it's handled in the FileSystemCache code instead). The only filesystem-related thing this test file deals with is the path used to create the cache(s), not the files themselves.